IntegrationTests/Services/AWSSTSIntegrationTests/STSWebIdentityAWSCredentialIdentityResolverTests.swift

```diff
@@ -42,13 +42,33 @@ class STSWebIdentityAWSCredentialIdentityResolverTests: XCTestCase {
     private let roleName = "aws-sts-integration-test-\(UUID().uuidString.split(separator: "-").first!.lowercased())"
     private let roleSessionName = "aws-sts-integration-test-\(UUID().uuidString.split(separator: "-").first!.lowercased())"
     private var roleArn: String!
+
     // JSON assume role policy
-    private let assumeRolePolicy = """
-    {"Version": "2012-10-17","Statement": [{"Sid": "","Effect": "Allow",
-    "Principal": {"Federated": "cognito-identity.amazonaws.com"},"Action": [
-    "sts:AssumeRoleWithWebIdentity"],"Condition": {"ForAnyValue:StringLike": {
-    "cognito-identity.amazonaws.com:amr": "unauthenticated"}}}]}
-    """
+    private var assumeRolePolicy: String {
+        return """
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "",
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Federated": "cognito-identity.amazonaws.com"
+                    },
+                    "Action": [
+                        "sts:AssumeRoleWithWebIdentity"
+                    ],
+                    "Condition": {
+                        "ForAnyValue:StringLike": {
+                            "cognito-identity.amazonaws.com:aud": "\(identityPoolId!)"
```

Comment:
Besides formatting, the only change to this policy is the line immediately above, where audience is used as the condition for assuming the role instead of authenticated / unauthenticated status.